components/src/core/DropdownToolbar/DropdownToolbar.tsx

```diff
@@ -0,0 +1,181 @@
+            paddingRight: theme.spacing(2),
+        },
+        navigationIconWrapper: {
+            marginRight: theme.spacing(4),
+            cursor: 'pointer',
+        },
+        title: {
+            lineHeight: 1,
+        },
+        subtitle: {
+            display: 'inline-flex',
+            cursor: 'pointer',
+        },
+        arrowDropdown: {
+            marginLeft: 4,
+        },
+        menuItem: {
+            height: MENU_ITEM_HEIGHT,
+        },
+    })
+);
+
+export type DropdownToolbarClasses = {
+    root?: string;
+    navigationIconWrapper?: string;
+    title?: string;
+    subtitle?: string;
+    arrowDropdown?: string;
+    menuItem?: string;
+};
+
+export const DropdownToolbar: React.FC<DropdownToolbarProps> = (props) => {
+    const { title, subtitleLabel, menuItems, navigationIcon, customMenu, style, classes } = props;
+    const [anchorEl, setAnchorEl] = useState(null);
+    const anchor = useRef(null);
+    const theme = useTheme();
+    const defaultClasses = useStyles(theme);
+    const getNavigationIcon = useCallback(() => {
+        if (navigationIcon) {
+            return (
+                <div className={clsx(defaultClasses.navigationIconWrapper, classes?.navigationIconWrapper)}>
+                    {navigationIcon}
+                </div>
+            );
+        }
+    }, [navigationIcon]);
+
+    return (
+        <>
+            <Toolbar data-testid="dropdown-toolbar" className={clsx(defaultClasses.root, classes?.root)} style={style}>
+                {getNavigationIcon()}
+                <ListItemText
+                    data-testid="dropdown-title"
+                    id={'dropdown-toolbar-text'}
+                    primary={
+                        <Typography variant="h6" className={clsx(defaultClasses.title, classes?.title)}>
+                            {title}
+                        </Typography>
+                    }
+                    secondary={
+                        <Typography
+                            id={'dropdown-subtitle'}
+                            data-testid={'dropdown-subtitle'}
+                            ref={anchor}
+                            aria-haspopup="true"
+                            className={clsx(defaultClasses.subtitle, classes?.subtitle)}
+                            component={'div'}
+                            onClick={(): void => {
+                                setAnchorEl(anchor.current);
+                            }}
+                        >
+                            {subtitleLabel || ''}
+                            {menuItems && menuItems.length > 0 && (
+                                <ArrowDropDown className={clsx(defaultClasses.arrowDropdown, classes?.arrowDropdown)} />
+                            )}
+                            {customMenu && (
+                                <ArrowDropDown className={clsx(defaultClasses.arrowDropdown, classes?.arrowDropdown)} />
+                            )}
```

Comment:
We only need one dropdown arrow
```suggestion
                            {((menuItems && menuItems.length > 0) || (customMenu)) && (
                                <ArrowDropDown className={clsx(defaultClasses.arrowDropdown, classes?.arrowDropdown)} />
                            )}
```